src/services/__tests__/profileService.test.ts

```diff
@@ -193,4 +209,16 @@ describe("ProfileService#upsertProfile", () => {
       expect(e).toEqual(new Error("Api error."));
     }
   });
+
+  it("fails to create-update a user profile returns an 429 HTTP error from the upstream API", async () => {
```

Comment:
fix sentence

Reply:
that sentence still doesn't make sense "it fails to create-update a user profile returns an 429 HTTP error from the upstream API"

Reply:
Fix in b2206e6